packages/network/src/logic/node/TrackerManager.ts

```diff
@@ -21,14 +21,14 @@ interface NodeDescriptor {
 }
 
 interface Subscriber {
-    subscribeToStreamIfHaveNotYet: (streamId: StreamIdAndPartition, sendStatus?: boolean) => void
+    subscribeToStreamIfHaveNotYet: (spid: SPID, sendStatus?: boolean) => void
```

Comment:
I'm a bit undecided if we should use `streamPartition` as names for these or `spid`?
```suggestion
    subscribeToStreamIfHaveNotYet: (streamPartition: SPID, sendStatus?: boolean) => void
```